example/docker-compose/local/tempo-local.yaml

```diff
@@ -28,6 +30,15 @@ compactor:
     block_retention: 1h
     compacted_block_retention: 10m
 
+metrics_generator:
+  registry:
+    external_labels:
+      source: docker-compose
```

Comment:
I'd consider setting this to `tempo` instead. Source imo is about the process generating the metrics and this makes it seem like docker-compose is emitting these metrics.

We could also do:
```
source: tempo
cluster: docker-compose
```